### PR TITLE
Add individual API proxy routes instead of bundled into one

### PR DESCRIPTION
**Pull request by cykoder**

### CODE REVIEW

File: pages/api/issue-biometric.js
Comment (mike-parkhill) [inline pages/api/issue-biometric.js:1]: isn't this just a special case of issue-credential? Does it need to be separate?

Comment (cykoder) [reply pages/api/issue-biometric.js:1]: separate to make wallet impl cleaner and easier, issue credential route requires entire credential payload. this mimicks what the wallet SDK already does

Comment (mike-parkhill) [reply pages/api/issue-biometric.js:1]: will this require a wallet update to match?

Comment (cykoder) [reply pages/api/issue-biometric.js:1]: this change doesn't depend on the wallet but rather the wallet depends on it

**Review by mike-parkhill — APPROVED** [review]

**Pull request merged**